Extensions/Notification/notificationtools.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @memberof gdjs.evtTools
+ * @namespace
+ * @static
+ * @public
```

Comment:
public is the default, you can remove it. static is for members of a class, you can remove it as well.

Reply:
This might sound dumb, but what does static mean in this situation?

Reply:
Oh okay thanks for the explanation.